hadoop-ozone/dist/src/main/smoketest/replication/wait.robot

```diff
@@ -17,7 +17,7 @@
 Documentation       Wait for replication to succeed
 Library             BuiltIn
 Resource            ../commonlib.robot
-Test Timeout        5 minutes
+Test Timeout        10 minutes
```

Comment:
This is because in acceptance test (secure) we launch 5 nodes to test EC file's disk usage, which requires more time for the robot test to complete, this specific test sometimes failed due to the time-out, so I increase the time-out here to allow it shut down gracefully~

Reply:
This issue has been addressed in another PR: https://github.com/apache/ozone/pull/3986

Reply:
Please remove this change from the PR.

Reply:
ohh!! yes thanks Atila!!